### ISSUE TITLE
About principle of activity regularizer in Keras

### ISSUE THREAD

**Issue by HyunkyuPark-biorobotics** [keras-team/keras#11465]
Dear keras users,

Hello everyone. I need some help about dealing with activity regularizer in keras.

As I know, the activity regularizer is the one that adding a 'norm of output' into a loss.

It is really similar to Tikhonov regularization, but is it really working well in the deep learning?

I doubt it because when differentiating the term of output norm loss with weight(w),  it becomes zero.

Actually I am not majored in Computer science, I know that I do not know that much.

So I want to ask some help know the basic principle of it.

Or, can you recommend me some related papers?

Thank you so much.
